htdocs/expedition/class/expedition.class.php

```diff
@@ -1724,12 +1724,12 @@ public function fetch_lines()
 
 			$shipmentlinebatch = new ExpeditionLineBatch($this->db);
 
+			$line = new ExpeditionLigne($this->db); // always set $line for PHAN analyser. @phan-var-force muse be used after an assignation, and there is no assignation for $line.
 			while ($i < $num) {
 				$obj = $this->db->fetch_object($resql);
 
-
 				if ($originline > 0 && $originline == $obj->fk_elementdet) {
-					'@phan-var-force ExpeditionLigne $line';  // $line from previous loop
+					// '@phan-var-force ExpeditionLigne $line'; // $line from previous loop
```

Comment:
@phan-var-force (l.1732) must be used [after an assignation](https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#inline-type-checks-via-string-literals), and we don't have any assignation for $line at the beginning of the loop.

Declaring $line before starting the loop is the solution I found to pass Phan tests.